Changes.md

```diff
@@ -18,6 +18,7 @@ Fixes
  - Fixed bug that caused the wrong plug to be edited by the Inspector with nested EditScopes.
  - Fixed bug that prevented selecting an Edit Scope that contained other Edit Scopes.
  - Fixed bug that caused an exception when simultaneously editing multiple plugs the Inspector.
+- NodeAlgo : Fixed bug that caused `isSetToUserDefault` to return `True` when a plug was connected to a compute node.
```

Comment:
Now 0.58.1.0 is released, this needs moving up into a new section.

Reply:
Fixed [here](https://github.com/GafferHQ/gaffer/pull/3895/commits/ca44cc782c17ed9bfaecc2f77c1a0769080eee14).